psp/captureauth_test.go

```diff
@@ -0,0 +1,51 @@
+package psp
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"testing"
+)
+
+func TestCapture(t *testing.T) {
```

Comment:
This should probably be called TestCaptureAuth

Reply:
done